test/Gas.test.js

```diff
@@ -99,7 +99,7 @@ contract("Gas costs", function (accounts) {
     describe("Check the gas cost for preCommit and commit with the identity proxy for ACTION key", async function () {
 
         const preCommitMaxGas = 95000;
-        const commitMaxGas = 85000;
+        const commitMaxGas = 175000;
```

Comment:
ganache improved gas estimates. Actual is '174100'

Reply:
Maybe we can add a variable `buffer`?
`const commitMaxGas = 174100 + buffer;`?

Then it is clear for everybody.